src/directive.js

```diff
@@ -2,7 +2,7 @@
  * Based on gg.editableText, originally created by Gabriel Grinberg on 6/13/14.
  */
 
-(function() {
+(function () {
```

Comment:
On purpose that this is differing from the previous style and the one we're using in the main repo?